src/plugins/ipmi_intf.c

```diff
@@ -219,7 +219,7 @@ ipmi_intf_session_set_username(struct ipmi_intf * intf, char * username)
 	if (!username)
 		return;
 
-	memcpy(intf->ssn_params.username, username, __min(strlen(username), 16));
+	memcpy(intf->ssn_params.username, username, __min(strlen(username), 16u));
```

Comment:
Just `16` without `u` is pretty well a `size_t` value. What's the need for `u` ? There were no warnings in ipmi_intf.c even without this change.

Reply:
Yes, it is not needed. It is just pedantically correct :)

Reply:
Let's not touch code without need. That hinders future blaming.